msal/src/MSAL.Common.props

```diff
@@ -0,0 +1,204 @@
+<Project>
+
+  <PropertyGroup>
+
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../../../build/MSAL.snk</AssemblyOriginatorKeyFile>
+    <DebugType>full</DebugType>
+    <!-- The MSAL.snk has both private and public keys -->
+    <DelaySign>false</DelaySign>
+    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)/../../build/SolutionWideAnalyzerConfig.ruleset</CodeAnalysisRuleSet>
+    <DefineConstants>$(DefineConstants);IS_MSAL</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Label="NuGet and AssemblyInfo metadata">
+    <!--This should be passed from the VSTS build-->
+    <AdalClientSemVer Condition="'$(MsalClientSemVer)' == ''">1.0.0-localbuild</AdalClientSemVer>
+    <!--This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion-->
+    <Version>$(MsalClientSemVer)</Version>
+
+    <!-- Copyright needs to be in the form of © not (c) to be compliant -->
+    <Title>Microsoft Authentication Library for .NET</Title>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Company>Microsoft Corporation</Company>
+    <Authors>Microsoft</Authors>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=844762</PackageLicenseUrl>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=844761</PackageProjectUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <Description>
+      This package contains the binaries of the Microsoft Authentication Library (MSAL).
+      MSAL makes it easy to obtain tokens from Azure AD v2 (work &amp; school accounts, MSA) and Azure AD B2C, gaining access to Microsoft Cloud API and any other API secured by Microsoft identities. This version supports adding authentication functionality to your .NET based client on Windows desktop (.NET 4.5+), UWP, .NET Core, Xamarin iOS and Xamarin Android.
+    </Description>
+    <PackageTags>Microsoft Authentication Library MSA MSAL B2C Azure Active Directory AAD Identity Authentication .NET Windows Store Xamarin iOS Android</PackageTags>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Company>Microsoft Corporation</Company>
+    <Product>Microsoft Authentication Library</Product>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PathToMsalSources>$(MSBuildThisFileDirectory)\Microsoft.Identity.Client</PathToMsalSources>
```

Comment:
I defined an extra 2 properties for paths so that they can be used from 2 different projects